src/level_three/advanced_settings/tasks/advanced_settings_tasks.c

```diff
@@ -156,6 +156,27 @@ void level_three_advanced_settings_tasks()
         tasks_read_card_id();
     } break;
 
+    case LEVEL_THREE_REGULATORY_INFO: {
+        instruction_content_t content[7] = {0};
+        LV_IMG_DECLARE(fcc_logo_black_2020);
+        LV_IMG_DECLARE(ce_mark);
+        content[0].img = &fcc_logo_black_2020;
+        content[0].img_x_offset = (128 - fcc_logo_black_2020.header.w) >> 1;
+        content[0].img_y_offset = 1;
+        content[0].text_align = LV_ALIGN_OUT_BOTTOM_MID;
+        content[6].img = &ce_mark;
+        content[6].img_x_offset = (128 - ce_mark.header.w) >> 1;
+        content[6].img_y_offset = 2;
+
+        snprintf(content[1].text, sizeof(content[1].text), "This device complies with Part 15 of the FCC Rules. Operation is");
```

Comment:
mark all irrelevant fields eg: `content[x].img = NULL` if we are not changing the structure

Reply:
Update: Calling memzero before populating data.